src/test/common_url_tests.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2024 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <test/util/setup_common.h>
```

Comment:
nit: It seems no symbols from that header are used in the source file.

Reply:
done